docs/service-catalog/docs/050-gs-broker-registration.md

```diff
@@ -0,0 +1,79 @@
+---
+title: How to register a broker
+type: Getting Started
+---
+
+This Getting Started guide shows how to register a new broker in the Service Catalog. The broker can be either a Namespace-scoped ServiceBroker or a cluster-wide ClusterServiceBroker. Follow the instructions based on the [UPS Broker example](https://github.com/kubernetes-incubator/service-catalog/tree/master/charts/ups-broker) to complete the guide.
+
+## Prerequisites
```

Comment:
do we really want to have such detailed prerequisites? SC is in Kyma, you do not have an option to decide on the version

Reply:
I've discussed this with @polskikiel who claims that it is really important to have a version of the SC that supports namespaced brokers. Also, after the modularization ( #809 ) the user will be able to choose a version.